build.sbt

```diff
@@ -203,7 +203,12 @@ def commonSettings: Seq[Setting[_]] = eclipseSettings ++ Seq(
 // host and port rather than looking it up from the service locator.
 // See docs/running-in-conductr.md for details.
 def kafkaSettings: Seq[Setting[_]] = Seq(
-  BundleKeys.startCommand += "-Dlagom.broker.kafka.service-name=''"
+  BundleKeys.startCommand ++= Seq(
+    "-Dlagom.broker.kafka.service-name=''",
+    // You may have to edit this list if your Kafka
+    // server is not listening on 127.0.0.1:9092
+    "-Dlagom.broker.kafka.brokers='127.0.0.1:9092'"
+  )
```

Comment:
Do you mean you're not sure if setting it in this particular ConductR setting is the recommended approach?

I'm not sure, either. When I asked about it in Slack, I didn't get any responses, and the Play guide offered a number of different options but didn't talk about ConductR specifically.

I based this approach on a little snippet of code I found in the [`sbt-conductr` README](https://github.com/typesafehub/sbt-conductr#adding-start-command-options). I'm not sure if this is best practice.

If it were more than a small number of properties, I would probably want to put them into a separate `.conf` file and pass `-Dconfig.resource` instead, as described in https://www.playframework.com/documentation/2.5.x/ProductionConfiguration#Specifying-an-alternate-configuration-file

I think #702 will be a better way to handle this in the long run.